src/send/EnterAmount.tsx

```diff
@@ -286,7 +299,10 @@ function EnterAmount({
     prepareTransactionsResult.transactions.length > 0
 
   const disabled =
-    disableProceed || (disableBalanceCheck ? !!tokenAmount?.isZero() : !transactionIsPossible)
+    disableProceed ||
+    (disableBalanceCheck
+      ? !!tokenAmount?.isZero() || (!!tokenAmount?.lte(token.balance) && !transactionIsPossible)
```

Comment:
We want to disable the continue button when checking if the prepared transaction is possible, even if we are disabling the warnings.

Reply:
I think the `!!tokenAmount?.isZero()` can be removed, because in that case `!!tokenAmount?.lte(token.balance) && !transactionIsPossible` should be true because 0 should be lte to the token balance and the transaction should not be possible.